CefSharp.Wpf/Experimental/WpfIMEKeyboardHandler.cs

```diff
@@ -61,7 +80,8 @@ public void ChangeCompositionRange(Range selectionRange, Rect[] characterBounds)
             {
                 //TODO: Getting the root window for every composition range change seems expensive,
                 //we should cache the position and update it on window move.
-                var parentWindow = Window.GetWindow(owner);
+                //In Winform embedded wpf borwser mode, Window.GetWindow(owner) is null, so use a custom function to get the outermost visual element.
+                var parentWindow = GetOutermostElement(owner);
```

Comment:
Please keep the existing behaviour.

If the `parentWindow` is null then fallback to calling `GetOutermostElement`